sortinghat/cmd/unify.py

```diff
@@ -23,13 +23,17 @@
 from __future__ import unicode_literals
 
 import argparse
+import json
+import os
 
 from .. import api
 from ..command import Command, CMD_SUCCESS, HELP_LIST
 from ..exceptions import MatcherNotSupportedError
 from ..matcher import create_identity_matcher, match
 from ..matching import SORTINGHAT_IDENTITIES_MATCHERS
 
+MATCHES_PATH = '~/.sortinghat/matches/matches.log'
```

Comment:
I don't think we need a subdir for this file. Actually, I'd rather name the archive to the command which is running it. In this case, `unify`.